es6-destructuring/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ES6 Destructuring</title>
+</head>
+<body>
+  <script>
+    const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+    let title = book1.title;
+    let author = book1.author;
+    let libraryID = book1.libraryID;
+
+    console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
+
+    const book2 = {
+      title: 'Les Fleurs du mal',
+      author: 'Charles Baudelaire',
+      libraryID: 2345
+};
+    title = book2.title;
+    author = book2.author;
+    libraryID = book2.libraryID;
+
+    console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
```

Comment:
Please destructure `book1` and `book2`:

---

![image](https://user-images.githubusercontent.com/16143771/174712038-87411bdf-2e4a-42e5-a716-8e97721479f4.png)

Reply:
![image](https://user-images.githubusercontent.com/97073517/174714781-c7586ebb-5bb9-491b-9b8a-90b6327d431c.png)
![image](https://user-images.githubusercontent.com/97073517/174714814-011d6000-9c51-42cb-89ed-1f1875362d88.png)